### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L vertexes,normaly`

### CODE REVIEW

File: PartOMagic/Base/FilePlant/xmls/Document_expressions.xml [hunk 1]
```diff
@@ -344,7 +344,7 @@
                 <Property name="CompoundTraversal" type="App::PropertyEnumeration" group="Lattice SubLink" doc="Sets how to unpack compounds if Looping is not &apos;Single&apos;." attr="0" ro="0" hide="0">
                     <Integer value="0" CustomEnum="true"/>
                     <CustomEnumList count="2">
-                        <Enum value="Compund children"/>
+                        <Enum value="Compound children"/>
```
Comment (luzpaz) [inline PartOMagic/Base/FilePlant/xmls/Document_expressions.xml]: please review carefully. Not sure if this type of change makes backward compatibility problems?

**Comment by luzpaz** [comment]
Soft bump :wink:

**Comment by DeepSOIC** [comment]
Sorry, I somehow didn't notice this PR altogether.

The errors in xmls are not Part-o-magic, they are from elsewhere (Lattice2, probably). These xmls are extracts from some FCStds, they are kept around for reference, and are not used by part-o-magic.

**Comment by DeepSOIC** [comment]
I'm too lazy now to merge just one commit, so I just merged it all =P

**Pull request merged**